src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -10377,6 +10444,63 @@ def pivot(
             sort=True,
         )
 
+    @register_query_compiler_method_not_implemented(
+        [None],
+        "pivot_table",
+        UnsupportedArgsRule(
+            unsupported_conditions=[
+                ("sort", False),
+                (
+                    lambda args: (
+                        args.get("index")
```

Comment:
```suggestion
                        args.get("index") is not None
```

This will otherwise pass over false-y values like `""` and `[]` (this may have been a bug in the original code).